doc/source/myrocks/limitations.rst

```diff
@@ -0,0 +1,53 @@
+.. _myrocks_limitations:
+
+===================
+MyRocks Limitations
+===================
+
+The MyRocks storage engine lacks the following features compared to InnoDB:

```

Comment:
There could be our/upstream bug reports discussing those limitations, especially the ones that are being planned to be lifted, would be nice to link there

Reply:
This is based on https://github.com/facebook/mysql-5.6/wiki/MyRocks-limitations
@georgelorch where can I get bug reports discussing the limitations?

Reply:
There really is no plan as we are at the mercy of upstream to decide if/when they will implement something or if we eventually decide to do it ourselves. There is no public roadmap.